app/policies/project_policy.rb

```diff
@@ -28,7 +28,7 @@ def resolve
       elsif is_admin?
         current_team.projects
       else
-        current_user.projects.not_archived.where(id: current_team.projects.pluck(:id))
+        current_user.projects.not_archived.where(id: current_team&.projects&.pluck(:id))
```

Comment:
@rands0n in this case it would be a `unless` instead of an `if`, but I agree it would improve readability 😄